torch/lib/THD/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 # TODO(jiayq): once we have unified CMake entry, remove this module path.
 SET(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake/Modules ${CMAKE_MODULE_PATH})
-SET(Gloo_USE_CUDA TRUE)
```

Comment:
this was a harmless line that didn't actually do anything. I checked downstream consequences of this, and there were actually none. Nice that you cleaned up / removed.

Reply:
as discussed IRL this wasn't harmless, this basically made sure FIND_PACKAGE(Gloo) returned true only when Gloo was compiled with CUDA.